### PR TITLE
fix: Twitter Cards meta tags

### PR DESCRIPTION
**Pull request by migueloller**

### CODE REVIEW

File: .changeset/nervous-suns-love.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (joshuawootonn) [inline .changeset/nervous-suns-love.md:1]: I am not sure how this works / where it is consumed so I can't really speak to it

File: packages/runtime/src/components/page/Page.tsx [hunk 2]
```diff
@@ -282,9 +282,9 @@ export function Page({ page, preview = false }: Props): JSX.Element {
         {socialImage && (
```
Comment (joshuawootonn) [inline packages/runtime/src/components/page/Page.tsx:282]: LGTM! 

Curious how you realized that this was a problem. Were you just sharing on twitter?

**Review by joshuawootonn — APPROVED** [review]

File: .changeset/nervous-suns-love.md (hunk 1 above)
Comment (joshuawootonn) [reply .changeset/nervous-suns-love.md:1]: Probably not a blocker though.

Comment (migueloller) [reply .changeset/nervous-suns-love.md:1]: This is consumed by our GitHub Actions workflow when we release a new version of the runtime

File: packages/runtime/src/components/page/Page.tsx (hunk 2 above)
Comment (migueloller) [reply packages/runtime/src/components/page/Page.tsx:282]: Andrew created an issue for it last week: https://linear.app/makeswift/issue/PRD-402/preview-card-isnt-displaying-in-twitter-when-sharing-a-makeswift-site

**Pull request merged**